internal/compiler/widgets/qt/spinner.slint

```diff
@@ -14,10 +17,20 @@ export component Spinner {
     accessible-role: progress-indicator;
     accessible-value: root.progress;
 
-    base := SpinnerBase {
-        width: 100%;
-        height: 100%;
-        stroke-width: 2px;
-        stroke:  NativeStyleMetrics.dark-color-scheme ? #60CDFF : #005FB8;
+    for i in count : Path {
+        stroke: black;
+        opacity: 0.9 - 0.8 * Math.mod(count / 2 + count * p + i , count ) / count ;
```

Comment:
This is what it looks like on `indeterminate` is `false` by `progress` is `1`.  I would suggest that all strokes that represents the current progress are displayed with opacity 0.9 and all that are not in the range of progress in 0.8 or less. So if you have progress == 1 all strokes are with opacity 0.9.

<img width="504" alt="image" src="https://github.com/slint-ui/slint/assets/6715107/6a185805-5a0a-4304-899a-6acbe44d8e34">